packages/components/src/placeholder/index.js

```diff
@@ -61,25 +87,27 @@ function Placeholder( {
 		'is-column-layout': isColumnLayout,
 	} );
 	return (
-		<div { ...additionalProps } className={ classes }>
-			{ resizeListener }
-			{ notices }
-			{ preview && (
-				<div className="components-placeholder__preview">
-					{ preview }
+		<AdminShadow>
```

Comment:
Any concern about a generic UI component (`@wordpress/components`) having knowledge about the existence of an "admin". 

The other question I have is why only apply this to placeholder component. From the `@wordpress/components` packages, it's just a component like any other component, how do you decide to apply this or not.

Reply:
Yeah, this is a work in progress, but I do think that this component is misplaced and belongs in the block editor package instead.

Reply:
(It's something to be used in block only.)

Reply:
> The other question I have is why only apply this to placeholder component. From the `@wordpress/components` packages, it's just a component like any other component, how do you decide to apply this or not.

It depends on whether the component is part of the editable content or not. Normally only placeholders are pieces of editor UI embedded in a block (instead of being rendered in the block toolbar or inspector).